requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-e .
+nose
+nose_fixes
+testfixtures
+manuel
+nose-cov
```

Comment:
Nah, please just use an extra_requires['test'] as in mush, it's the pattern I use in all my projects...